exonum/src/api/node/public/explorer.rs

```diff
@@ -297,6 +280,22 @@ impl ExplorerApi {
         });
     }
 
+    fn tx_handler(
+        state: &ServiceApiState,
+        query: TransactionHex,
+    ) -> Result<TransactionResponse, ApiError> {
+        let buf: Vec<u8> = ::hex::decode(query.tx_body).map_err(into_failure)?;
```

Comment:
This code is very similar to `impl Handler<Transaction> for Server` in the `websocket` module. Is it possible to deduplicate it?